Marlin/language_de.h

```diff
@@ -35,37 +35,37 @@
 //#define SIMULATE_ROMFONT
 #define DISPLAY_CHARSET_ISO10646_1
 
-#define WELCOME_MSG                         MACHINE_NAME " bereit."
-#define MSG_SD_INSERTED                     "SDKarte erkannt."
-#define MSG_SD_REMOVED                      "SDKarte entfernt."
+#define WELCOME_MSG                         MACHINE_NAME " bereit"
+#define MSG_SD_INSERTED                     "SD-Karte erkannt"
+#define MSG_SD_REMOVED                      "SD-Karte entfernt"
 #define MSG_MAIN                            "Hauptmenü"
 #define MSG_AUTOSTART                       "Autostart"
-#define MSG_DISABLE_STEPPERS                "Motoren Aus" // M84
+#define MSG_DISABLE_STEPPERS                "Motoren aus" // M84
 #define MSG_AUTO_HOME                       "Home" // G28
 #define MSG_LEVEL_BED_HOMING                "Homing XYZ"
 #define MSG_LEVEL_BED_WAITING               "Klick für Start"
 #define MSG_LEVEL_BED_DONE                  "Fertig"
 #define MSG_LEVEL_BED_CANCEL                "Abbruch"
-#define MSG_SET_HOME_OFFSETS                "Setze Home hier"
+#define MSG_SET_HOME_OFFSETS                "Setze H.-Offsets"
 #define MSG_HOME_OFFSETS_APPLIED            "Offsets aktiv"
-#define MSG_SET_ORIGIN                      "Setze Null hier" //"G92 X0 Y0 Z0" commented out in ultralcd.cpp
+#define MSG_SET_ORIGIN                      "Setze Nullpunkt" //"G92 X0 Y0 Z0" commented out in ultralcd.cpp
 #define MSG_PREHEAT_PLA                     "Vorwärmen PLA"
 #define MSG_PREHEAT_PLA_N                   "Vorwärmen PLA "
 #define MSG_PREHEAT_PLA_ALL                 "Vorw. PLA Alle"
 #define MSG_PREHEAT_PLA_BEDONLY             "Vorw. PLA Bett"
-#define MSG_PREHEAT_PLA_SETTINGS            "Vorwärm. PLA Ein."
+#define MSG_PREHEAT_PLA_SETTINGS            "Vorw. PLA Einst."
 #define MSG_PREHEAT_ABS                     "Vorwärmen ABS"
 #define MSG_PREHEAT_ABS_N                   "Vorwärmen ABS "
 #define MSG_PREHEAT_ABS_ALL                 "Vorw. ABS Alle"
 #define MSG_PREHEAT_ABS_BEDONLY             "Vorw. ABS Bett"
-#define MSG_PREHEAT_ABS_SETTINGS            "Vorwärm. ABS Ein."
+#define MSG_PREHEAT_ABS_SETTINGS            "Vorw. ABS Einst."
 #define MSG_COOLDOWN                        "Abkühlen"
-#define MSG_SWITCH_PS_ON                    "Netzteil Ein"
-#define MSG_SWITCH_PS_OFF                   "Netzteil Aus"
+#define MSG_SWITCH_PS_ON                    "Netzteil ein"
+#define MSG_SWITCH_PS_OFF                   "Netzteil aus"
 #define MSG_EXTRUDE                         "Extrudieren"
 #define MSG_RETRACT                         "Retract"
 #define MSG_MOVE_AXIS                       "Bewegen"
-#define MSG_LEVEL_BED                       "Bett Korrektur"
+#define MSG_LEVEL_BED                       "Bett vermessen"
```

Comment:
das korrekte Wort ist nicht "vermessen", sonder "nivellieren":

```
01234567890123456789
Bett nivellieren
```

passt dann in 16 Zeichen